torch/_torch_docs.py

```diff
@@ -2871,6 +2871,25 @@ def merge_dicts(*dicts):
     tensor([False, True, False])
 """)
 
+add_docstr(torch.isreal,
+           r"""
+isreal(input) -> (bool)
+
+Returns True if the data type of :attr:`input` is a complex data type i.e.,
```

Comment:
See the documentation for torch.isnan for an idea of how to template this (https://pytorch.org/docs/master/generated/torch.isnan.html). Basically it returns a tensor containing True wherever the value in input is real-valued, which is all real-valued types and complex values whose imaginary part is zero.

Reply:
Ex. "Returns a new tensor with boolean elements representing if each element of input is real-valued or not..."